spark/interpreter/src/main/java/org/apache/zeppelin/spark/OldSparkInterpreter.java

```diff
@@ -1041,8 +1041,7 @@ public InterpreterResult interpret(String[] lines, InterpreterContext context) {
     synchronized (this) {
       z.setGui(context.getGui());
       z.setNoteGui(context.getNoteGui());
-      String jobDesc = "Started by: " + Utils.getUserName(context.getAuthenticationInfo());
-      sc.setJobGroup(Utils.buildJobGroupId(context), jobDesc, false);
+      sc.setJobGroup(Utils.buildJobGroupId(context), Utils.buildJobDesc(context), false);
```

Comment:
should check if buildJobDesc returns nothing? anonymous?

Reply:
It won't be null even it is anonymous, https://github.com/apache/zeppelin/pull/3044/files#diff-25ff02ea9a7b4a31efaf02e8b401ea5bR156